docs/server/settings.rst

```diff
@@ -294,20 +298,50 @@ Translation environment configuration settings.
             'HOST': 'localhost',
```

Comment:
Reason to keep is to keep backwards compatibility and renaming to `local` actually doesn't contribute anything.